spring-cloud-contract-verifier/src/test/java/org/springframework/cloud/contract/verifier/util/ContractVerifierUtilTest.java

```diff
@@ -0,0 +1,127 @@
+package org.springframework.cloud.contract.verifier.util;
+
+import org.junit.Test;
+import org.w3c.dom.Document;
+import org.w3c.dom.Node;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.io.StringReader;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class ContractVerifierUtilTest {
+
+	private final String unnamedXml = "<customer>\n" +
+			"      <email>customer@test.com</email>\n" +
+			"    </customer>";
+
+	private final String namedXml = "<ns1:customer xmlns:ns1=\"http://demo.com/testns\">\n" +
+			"      <email>customer@test.com</email>\n" +
+			"    </ns1:customer>";
+
+	@Test
+	public void shouldGetValueFromXPath() throws ParserConfigurationException, IOException, SAXException {
+		//Given
+		DocumentBuilderFactory builderFactory = DocumentBuilderFactory.newInstance();
+		builderFactory.setNamespaceAware(true);
+		DocumentBuilder documentBuilder = builderFactory.newDocumentBuilder();
+		Document parsedXml = documentBuilder.parse(new InputSource(new StringReader(unnamedXml)));
+		//When
+		String value = ContractVerifierUtil.valueFromXPath(parsedXml, "/customer/email/text()");
+		//Then
+		assertEquals("customer@test.com", value);

```

Comment:
Can we use AssertJ here instead of junit assertions?

Reply:
Done

Reply:
done